src/vs/editor/common/viewModel/monospaceLineBreaksComputer.ts

```diff
@@ -353,13 +357,17 @@ function createLineBreaksFromPreviousLineBreaks(classifier: WrappingCharacterCla
 	return previousBreakingData;
 }
 
-function createLineBreaks(classifier: WrappingCharacterClassifier, lineText: string, tabSize: number, firstLineBreakColumn: number, columnsForFullWidthChar: number, wrappingIndent: WrappingIndent): LineBreakData | null {
+function createLineBreaks(classifier: WrappingCharacterClassifier, _lineText: string, injectedTexts: LineInjectedText[] | null, tabSize: number, firstLineBreakColumn: number, columnsForFullWidthChar: number, wrappingIndent: WrappingIndent): LineBreakData | null {
+	const lineText = LineInjectedText.applyInjectedText(_lineText, injectedTexts);
+
 	if (firstLineBreakColumn === -1) {
+		// TODO: return LineBreakData that only has injected text if necessary
```

Comment:
We need to tackle this TODO before merging. We need to support injected text when wrapping is disabled. We have to see if `LineBreakData` has any assumptions about the `breakingOffsets.length` being `> 1` and then construct here a `LineBreakData` which basically has no breaking offsets. I believe in that case `breakingOffsets[0]` should point to the line text length.

But we should not let this `if` statement fall through. i.e. the algorithm below is not prepared for `firstLineBreakColumn` to be `-1`.